sahi/utils/cv.py

```diff
@@ -380,7 +380,7 @@ def visualize_object_predictions(
         Path(output_dir).mkdir(parents=True, exist_ok=True)
         # save inference result
         save_path = os.path.join(output_dir, file_name + "." + export_format)
-        cv2.imwrite(save_path, cv2.cvtColor(image, cv2.COLOR_RGB2BGR))
```

Comment:
We should not remove this line because it is necessary in `export_visual()` method of the `PredictionResult` class: https://github.com/obss/sahi/blob/2824b98c344e052289343cf1a4af23c477325454/sahi/prediction.py#L167

If you dont want output, you can call `visualize_object_predictions` with visualize_object_predictions(output_dir=None)

Reply:
If I don't remove this line, each video frame saving as image in `runs/exp/visual` during `export_visual` and `export_video`. But, if I want to inference on image with `export_visual`, I save result of prediction image in [sahi/sahi/predict.py (line:709)](https://github.com/madenburak/sahi/blob/2824b98c344e052289343cf1a4af23c477325454/sahi/predict.py#L709).

Reply:
Lines between 381-383 should not be deleted in `sahi/utils/cv.py`. It is necessary for other functions and classes.

Instead we should add a check in `predict` function so that `output_dir=None` is set for `visualize_object_predictions` function when `export_video=True` and `export_visual=True`

Reply:
@madenburak these changes should be reverted